joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitor.scala

```diff
@@ -660,7 +660,8 @@ class PythonAstVisitor(
           "",
           lineAndColumn,
           arguments,
-          keywordArguments
+          keywordArguments,
+          ""
```

Comment:
Once you use the `Option[String] = None` parameter, you don't need to add all of these blank entries